vkcs/internal/clients/clients.go

```diff
@@ -41,3 +59,57 @@ func newMLPlatformV1(client *gophercloud.ProviderClient, eo gophercloud.Endpoint
 	sc.ResourceBase = sc.Endpoint + "v1_0/"
 	return sc, err
 }
+
+func newCapabilityV1(client *gophercloud.ProviderClient, eo gophercloud.EndpointOpts) (*gophercloud.ServiceClient, error) {
+	baseURL, err := findBaseURL(client, eo)
+	if err != nil {
+		return nil, err
+	}
+
+	sc := new(gophercloud.ServiceClient)
+	eo.ApplyDefaults("ics")
+	sc.ProviderClient = client
+	sc.Type = "ics"
+
+	sc.Endpoint = fmt.Sprint(baseURL, "infra/ics/")
+	sc.ResourceBase = sc.Endpoint + "v1/"
+
+	return sc, nil
+}
+
+func newCloudMonitoringV2(client *gophercloud.ProviderClient, eo gophercloud.EndpointOpts) (*gophercloud.ServiceClient, error) {
+	baseURL, err := findBaseURL(client, eo)
+	if err != nil {
+		return nil, err
+	}
+
+	sc := new(gophercloud.ServiceClient)
+	eo.ApplyDefaults("templater")
+	sc.ProviderClient = client
+	sc.Type = "templater"
+
+	sc.Endpoint = fmt.Sprint(baseURL, "infra/templater/")
+	sc.ResourceBase = sc.Endpoint + "v2/"
+
+	return sc, nil
+}
+
+func findBaseURL(client *gophercloud.ProviderClient, eo gophercloud.EndpointOpts) (string, error) {
+	eo.ApplyDefaults("data-protect")
+	url, err := client.EndpointLocator(eo)
+	if err != nil {
+		var errNotFound *gophercloud.ErrEndpointNotFound
+		if errors.As(err, &errNotFound) {
+			return "", NewErrEndpointNotFound("Cloud monitoring")
```

Comment:
Okay, I changed my mind. Let's just return an original error, f*k this.

Reply:
change service name to data-protect, we also discussed about this with @ftersin